src/svc/fluxdocsd/schemas/fluxdocs.yml

```diff
@@ -5,6 +5,7 @@ required:
   - kind
   - headline
   - fluxType
+  - example
```

Comment:
I don't think we need this on the backend. Let's make it optional rather than required by removing it from the required list.

Reply:
is there going to be a case where a flux function will not have an example value? it seems like they all should have this property..